hoomd/hpmc/ComputeFreeVolume.h

```diff
@@ -303,9 +320,33 @@ Scalar ComputeFreeVolume<Shape>::getLogValue(const std::string& quantity, unsign
     throw std::runtime_error("Undefined log quantity");
     }
 
+// \return the free volume.
+template<class Shape>
+Scalar ComputeFreeVolume<Shape>::getFreeVolume()
```

Comment:
The way this is written and implemented, it seems that we will never from Python run `computeFreeEnergy`. Is there a reason we have separated the computation from the accessing of the computed quantity?

Reply:
I copied the code from the `getLogValue` method into the `getFreeVolume` method to have the Python log `free_volume` method return a `free_volume` pybind property, because it looks like that was what was done in `md/ComputeThermo.h` and `compute.ThermodynamicQuantities`, I can change it to directly call `computeFreeVolume` from Python though

Reply:
`computeFreeVolume` is the internal implementation of the method called at the appropriate time in `compute`. 

`computeFreeVolume` should be a `protected` member. The method that Python needs to call is `compute`, which @danevans09 calls in the `free_volume` property.